mrp_production_partner_note/models/procurement_order.py

```diff
@@ -0,0 +1,19 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+
+from openerp import models, api
+
+
+class ProcurementOrder(models.Model):
+    _inherit = 'procurement.order'
+
+    @api.model
+    def _prepare_mo_vals(self, procurement):
+        res = super(ProcurementOrder, self)._prepare_mo_vals(procurement)
+        sale_proc = procurement.move_dest_id.procurement_id
+        mrp_notes = sale_proc.sale_line_id.order_id.partner_id.mrp_notes
+        if mrp_notes:
+            res['notes'] = mrp_notes
```

Comment:
I personally do not like sum strings.

<<This is a personal preference>> 
I prefer the format instead (but it is a personal preference to avoid any "type problem").

I prefer format or concatenation instead the (+) but it is ok.